tests/integration/test_ui_integration.py

```
@@ -0,0 +1,135 @@
+"""Integration tests for UI components with the main game loop."""
+import unittest
+from unittest.mock import MagicMock, patch
+
+import pygame
+
+from src.config import display
```

Ruff (0.8.2)

`src.config.display` imported but unused

Remove unused import: `src.config.display`

(F401)